runtime/compiler/optimizer/InlinerTempForJ9.cpp

```diff
@@ -831,11 +831,14 @@ TR_J9InlinerPolicy::createAnchorNodesForUnsafeGetPut(TR::TreeTop* treeTop,
 
       TR::Node* node = treeTop->getNode();
       TR::TreeTop *compRefTT =
-         TR::TreeTop::create(comp(), genCompressedRefs(isUnsafeGet ? node->getFirstChild() : node,
-                                                       false));
-      TR::TreeTop *prevTT = treeTop->getPrevTreeTop();
-      prevTT->join(compRefTT);
-      compRefTT->join(isUnsafeGet ? treeTop : treeTop->getNextTreeTop());
+         TR::TreeTop::create(comp(), genCompressedRefs(isUnsafeGet?node->getFirstChild():node,
+                                                      false));
+      if (compRefTT)
```

Comment:
do you ever expect compRefTT can be NULL? it came from `create()` ... if it failed to create, compilation was expected to be aborted before return, I believed.